pcdsdevices/epics_motor.py

```diff
@@ -197,6 +285,26 @@ def _get_epics_limits(self) -> tuple[float, float]:
             return (0, 0)
         return limits
 
+    @property
+    def msta(self):
+        """
+        Returns the msta fields as a dictionary.
+        """
+        # the MSTA field is a float for some reason...
+        val = int(self.msta_raw.get())
```

Comment:
This is so incredibly puzzling to me. The manual says that it's supposed to be a ULONG but it's clearly a DOUBLE if you check any of the fields...

Reply:
This isn't something we did specific to our records right?...

Reply:
It seems to be universal between standard records and the weird stuff we created here. I can't rule out a strange custom motor record build I guess.

Reply:
Looking at our latest motor record (R6.10.1-2.3.3), it's a DBF_ULONG in motorRecord.dbd as well as in e.g. ims.dbd, so I'm not sure why it's a float. 
![image](https://github.com/user-attachments/assets/7f3794e9-ea60-4fa5-bfdd-c753f2af3ca9)
![image](https://github.com/user-attachments/assets/a0ce6661-b586-442b-b058-a87efadfe289)